src/Analysis/Ast/Test/TypeshedTests.cs

```diff
@@ -45,7 +45,7 @@ import sys
 e1, e2, e3 = sys.exc_info()
 ";
             var analysis = await GetAnalysisAsync(code);
-            // sys.exc_info() -> (exception_type, exception_value, traceback)
+            // def exc_info() -> Tuple[Optional[Type[BaseException]], Optional[BaseException], Optional[TracebackType]]: ...
```

Comment:
A note to check https://github.com/microsoft/python-language-server/issues/1371#issuecomment-515232161 with this PR merged. The cumulative fixes might make typeshed updatable.